classes/issuer_management.php

```diff
@@ -65,6 +62,13 @@ public static function is_valid_issuer(\core\oauth2\issuer $issuer) {
         return $endpointtoken && $endpointauth && $endpointuserinfo;
     }
 
+    /**
+     * If a webfinger URL is set for the oauth service/issue return it, otherwise return FALSE
+     *
+     * @param \core\oauth2\issuer $issuer
+     * @return string|bool
```

Comment:
I suppose that you have to write these lines to meet the moodle code-style standard.
It adds no value, because the parameter and return types are declared exactly the same in the real code below.

Reply:
yes, at least the `@param` is compulsory